test/fixtures/server-startup-control/delayed-compilation.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+const {execSync} = require('child_process');
+
+module.exports = () => {
+  return {
+    visitor: {
+      StringLiteral(path /* : any */) {
+        if (path.node.value === 'TRIGGER-BABEL-DELAY') {
+          console.log('Triggered babel delay');
+          execSync('sleep 3');
```

Comment:
Should probably use node-IPC or something for more robust/faster tests, but this is a lot less code. We can update this if we see flakiness on CI.